gnocchi/storage/s3.py

```diff
@@ -50,6 +50,11 @@
                  help="Maximum time to wait checking data consistency when "
                  "writing to S3. Set to 0 to disable data consistency "
                  "validation."),
+    cfg.IntOpt('s3_max_pool_connections',
+                min=10,
```

Comment:
I this really 10? Seems to me that 1 should work.

Reply:
@jd  10 is de default value that comes on the adapter.py of the python requests library, i really think lowering it is not a good call: https://github.com/requests/requests/blob/master/requests/adapters.py#L47

Reply:
It's not saying it's a good idea. It's saying it's possible. That's what this min/max values need to represent.

The _default_ value is what should be a good idea. :)